src/__internal__/validation-message/validation-message.style.ts

```diff
@@ -2,17 +2,22 @@ import styled, { css } from "styled-components";
 
 interface StyledValidationMessageProps {
   isWarning?: boolean;
+  isDarkBackground?: boolean;
 }
 
 const StyledValidationMessage = styled.p<StyledValidationMessageProps>`
-  ${({ isWarning }) => css`
-    color: ${isWarning
-      ? "var(--colorsSemanticCaution600)"
-      : "var(--colorsSemanticNegative500)"};
-    font-weight: ${isWarning ? "normal" : "500"};
-    margin-top: 0px;
-    margin-bottom: 8px;
-  `}
+  ${({ isWarning, isDarkBackground }) => {
+    const darkBgColour =
+      !isWarning && isDarkBackground
```

Comment:
comment: we can probably remove the `isWarning` check here as it's checked on line 15